### ISSUE TITLE
Does it have keep alive by default?

### ISSUE THREAD

**Issue by vvo** [koichik/node-tunnel#23]
Hi! I am trying to understand if when I use this agent it'll have keep alive enabled? Thanks

**Comment by KiraYo4kage** [comment]
I have the same question~

**Comment by cloudhary** [comment]
I'm curious too. I'm currently investigating this. @vvo @KiraYo4kage did either of you manage to figure it out?

**Comment by vvo** [comment]
nope but @koichik might know? Maybe our question is also not relevant, let us know!

**Comment by flotwig** [comment]
This library doesn't support Keep-Alive, but https://github.com/mknj/node-keepalive-proxy-agent seems to support it!

**Comment by omeshp** [comment]
No, it doesn't. I have a pending PR https://github.com/koichik/node-tunnel/pull/36/files#diff-7cb25e6733372586a7d0530feba6f8aeR72 which adds this support.
